test/integration/network.js

```diff
@@ -95,11 +95,11 @@ describe("Network", function() {
   });
 
   it("gets invalid json message", ()=>{
-    return network.httpFetch("www.google.com.br")
+    return network.httpFetch("http://www.google.com.br")
       .then((resp)=>{
         return resp.json();
       }).catch((error)=>{
-         assert.equal(error.message, "invalid json response body at www.google.com.br reason: Unexpected token < in JSON at position 0");
+         assert(error.message.includes, "invalid json response body");
```

Comment:
@tejohnso shouldn't this be `assert(error.message.includes("invalid json response body"));` ?

Reply:
Definitely. Good :eye: .  See latest commit.